cli/lib/kontena_cli.rb

```diff
@@ -87,6 +109,20 @@ def self.cli_root(*joinables)
       File.join(Kontena.root, 'lib/kontena/cli', *joinables)
     end
   end
+
+  def self.logger
+    return @logger if @logger
+    require 'logger'
+    @logger = Logger.new($LOG_TARGET, 'weekly')
+    @logger.level = ENV["DEBUG"] ? Logger::DEBUG : Logger::INFO
+    @logger.progname = 'CLI'
```

Comment:
That would initialize a bunch of extra loggers writing to the same log device. Logger is only thread safe with one instance.